lib/jekyll/collection.rb

```diff
@@ -64,8 +65,10 @@ def read
             Jekyll.logger.debug "Skipped From Publishing:", doc.relative_path
           end
         else
-          relative_dir = Jekyll.sanitized_path(relative_directory, File.dirname(file_path)).chomp("/.")
-          files << StaticFile.new(site, site.source, relative_dir, File.basename(full_path), self)
+          relative_dir = Jekyll.sanitized_path(relative_directory,
+            File.dirname(file_path)).chomp("/.")
+          files << StaticFile.new(site, site.source, relative_dir,
+            File.basename(full_path), self)
```

Comment:
If a short line like this has to be sent to a new line our new trial is to try and collapse the class, (assuming since you made the "noob comment") if you look at the top you can see all those classes so collapse them from:

```ruby
module Jekyll
  class MyClass1
    class MyClass2
      class MyClass3
      end
    end
  end
end
```

Into:

```ruby
class Jekyll::MyClass1::MyClass2
   class MyClass3
   end
end
```

And then go from there and see how much you have to shrink.  We lose a lot of space in Jekyll so we are trying to regain that space to keep readability high, at least this is what I remember being agreed upon yeah @jekyll/core?